raiden/network/proxies/token_network.py

```diff
@@ -305,15 +327,18 @@ def _new_netting_channel(
             settle_timeout=settle_timeout,
         )
         if not gas_limit:
+            newest_block = self.client.blockhash_from_blocknumber("latest")
             self.proxy.jsonrpc_client.check_for_insufficient_eth(
                 transaction_name="openChannel",
                 transaction_executed=False,
                 required_gas=self.gas_measurements["TokenNetwork.openChannel"],
                 block_identifier=checking_block,
```

Comment:
For clarify, use `newest_block` here instead of `checking_block`

Reply:
I tried that and it crashed in the integration tests with `Could not encode to JSON`. The problem is `web3.eth.getBalance` being called with a (binary) block hash, converting the block hash to hex doesn't work either (`ValueError: {'code': -32602, 'message': 'invalid argument 1: hex number > 64 bits'}`) I don't know how to fix it, apparently `web3.eth.getBalance` doesn't expect a block hash at all but rather a block number?